src/schinkenfleckerl-gratinated.md

```diff
@@ -1,5 +1,9 @@
 # Schinkenfleckerl überbacken, Viennese Mac'n'Cheese
 
```

Comment:
Can you add `.webp` image here?

Reply:
I'd love to but plan to cook it in a week or two and have no pictures atm.